gulpfile.js

```diff
@@ -1,32 +1,27 @@
-var gulp = require( 'gulp' );
-var merge = require('merge-stream');
-var del = require( 'del' );
-var run = require( 'gulp-run' );
-var replace = require( 'gulp-string-replace' );
-
-var fs = require( 'fs' );
-var config = JSON.parse( fs.readFileSync( './package.json' ) );
+const del = require( 'del' );
+const fs = require( 'fs' );
+const gulp = require( 'gulp' );
+const run = require( 'gulp-run' );
 
 gulp.task( 'verify:versions', function () {
 	return run( 'php bin/verify-versions.php' ).exec();
-} )
-
-gulp.task( 'version', function () {
-	var pluginStream = gulp.src( [ 'genesis-custom-blocks.php' ] )
-		.pipe( replace( new RegExp( /Version:\s*(.*)/, 'g' ), "Version: " + config.version ) )
```

Comment:
We're manually setting the version in `genesis-custom-blocks.php`, so no need for this